index.js

```diff
@@ -290,6 +291,16 @@ class HomeBridgeTasmotaAirconHTTP {
       Vendor: state.vendor,
     };
   }
+
+  _valueOfFanForTasmota() {
+   if (this.state.fanAutoSpeed == 'low') {
+     return String(Math.floor((this.fanSteps + 1) * this.state.fanSpeed / 101) || 'auto');
+   } else if (this.state.fanAutoSpeed == 'force') {
+     return 'auto';
+   } else {
+     return String(Math.floor(this.fanSteps * this.state.fanSpeed / 101) + 1);
```

Comment:
If the slider is at max level (100), the calculated level will be equal to fan `fanSteps + 1` if i divide by 100. By dividing by 101, I'm sure the level will never be higher than `fanSteps`.

I agree this is not easy to understand at first see. Still searching for a better way to calculate this. But because the slider start from 0 to 100, there is 101 positions.

Reply:
What was wrong with the original calculation?
`Math.ceil(this.fanSteps * state.fanSpeed / 100) || 1`